packages/components/date-picker/src/date-picker.tsx

```diff
@@ -61,7 +61,7 @@ export default defineComponent({
 
     expose(refProps)
 
-    const onModelValueUpdated = (val: any) => {
+    const onModelValueUpdated = (val: Date) => {
```

Comment:
I feel that the types here are not just `Date`, they may be closer to `SingleOrRange<DateModelType> | null`

Reply:
Okay, I modified it.